### ISSUE TITLE
Visual indicator for price compared to market history

### ISSUE THREAD

**Issue by rm-you** [jamesjamail/TunnelQuestBot#30]
If there is a parsed price value, and there is historical wiki data, we could display a quick visual indicator for whether something is a good deal or not!

For example:

Below the 30-day range: 🔥
Inside the 30-day range: ✔️
Above the 30-day range: 💩

**Comment by jamesjamail** [comment]
Pretty cool idea.  I like it! I'm a bigger fan of data-esque emojis, the poop and fire seem kind of silly to me.

**Comment by rm-you** [comment]
Yeah, I was looking for ones like a stock market chart trending up or down, and I found a couple, but every time I looked away and back, I couldn't remember which meant Expensive and which meant Cheap.
PLEASE help me find better ones! The only requirement I think is that they're standard UTF-8 emoji so we don't have to rely on anything else. ALTERNATIVELY we could use discord emoji, and just include the PNG files in the repo and maybe even have the bot upload them to the server, if that's possible.
